cpp/src/io/parquet/page_decode.cuh

```diff
@@ -1387,7 +1389,7 @@ inline __device__ bool setupLocalPageInfo(page_state_s* const s,
 
       // if we're in the decoding step, jump directly to the first
       // value we care about
```

Comment:
I think this comment needs to be updated.

Reply:
Could change this to move the `is_bonds_step` test inside the `else` block...I think all that's necessary is to just not zero out those values.